IO/ConsoleUtil.cs

```diff
@@ -39,41 +37,15 @@ public static void Write(string content, ConsoleColor color = ConsoleColor.White
 
                 Console.Write(content);
 
-                Console.ForegroundColor = ConsoleColor.White;
-                Console.BackgroundColor = ConsoleColor.Black;
+                Console.ResetColor();
 
-                //Logger.Log(content);
+                Logger.Log(content);
             }
         }
 
         public static void WriteLine(string content, ConsoleColor color = ConsoleColor.White, int height = 0)
         {
-            lock (_lck)
-            {
-                content = content.Trim().Trim(ToTrim);
-                content = string.IsNullOrEmpty(content) ? string.Empty : $"[{DateTime.Now:yyyy-MM-dd HH:mm:ss.fff zzz}] {content}";
-
-                Console.BackgroundColor = ConsoleColor.Black;
-
-                try
-                {
-                    Console.ForegroundColor = color;
-                }
-                catch
-                {
-                    Console.ForegroundColor = ConsoleColor.White;
-                }
-
-                if (height > 0)
-                    Console.CursorTop += height;
-
-                Console.WriteLine(content);
-
-                Console.ForegroundColor = ConsoleColor.White;
-                Console.BackgroundColor = ConsoleColor.Black;
-
-                //Logger.Log(content);
-            }
+            Write(content + Environment.NewLine, color, height);
```

Comment:
I'd consider using WriteLine instead of Write text + newline in case the implementation differs on some platforms

Reply:
Look at implementation that was commented before, I didn't want to make it redundant, should I use bool parameter on end of Write method or what?